packages/grafana-ui/src/components/Icon/Icon.tsx

```diff
@@ -67,6 +67,7 @@ export const Icon = React.forwardRef<SVGElement, IconProps>(
         title={title}
         className={composedClassName}
         style={style}
+        icon-size={size}
```

Comment:
Is this a valid attribute to pass to the SVG component or should the icon size be converted to the correct styles?